### ISSUE TITLE
Bug residual bottleneck unit implementation in slim. 

### ISSUE THREAD

**Issue by zeyademam** [tensorflow/models#5198]
### System information
- **What is the top-level directory of the model you are using**: N/A
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: N/A
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: N/A
- **TensorFlow installed from (source or binary)**: N/A
- **TensorFlow version (use command below)**: N/A
- **Bazel version (if compiling from source)**: N/A
- **CUDA/cuDNN version**: N/A
- **GPU model and memory**: N/A
- **Exact command to reproduce**: N/A

------------------------
### Problem Description
In the documentation of the `bottleneck()` function here: https://github.com/tensorflow/models/blob/master/research/slim/nets/resnet_v2.py

It says the bottleneck unit is a direct adaptation of figure 1)(b) here: https://arxiv.org/pdf/1603.05027.pdf

However, there are differences, for instance, the code has only one batch norm layer whereas the figure has two.

**Comment by zeyademam** [comment]
@bitfort any luck comments on this?

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
